src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -128,6 +129,19 @@ string GetBookmarkFromUid(string uid)
                 => Regex.Replace(uid, @"\W", "_");
         }
 
+        private static (Error, (bool isRoot, string jsonPath, Dictionary<string, Lazy<JToken>> propertiesByUid)) AggregateXrefSpecProperties(string uid, List<(bool isRoot, string jsonPath, Dictionary<string, Lazy<JToken>> properties)> uidWithProperties)
+        {
+            if (uidWithProperties.Count > 1)
+            {
+                return (Errors.UidConflict(uid), uidWithProperties.OrderBy(x => x.jsonPath).First());
```

Comment:
prefer order by sourceinfo over json path because json path order is not from top to bottom